### ISSUE TITLE
Starting E2E tests sometime fail because eth node not available

### ISSUE THREAD

**Issue by lumtis** [zeta-chain/node#2969]
**Describe the Issue**

The E2E test setup sometimes fails with the following error message:

```
Fatal: Failed to start the JavaScript console: api modules: Post "http://eth:8545": dial tcp: lookup eth on 127.0.0.11:53: no such host
setup      | starting E2E tests
Unable to continue execution: failed to get clients from config: failed to get evm client: failed to get chain id: Post "http://eth:8545": dial tcp: lookup eth on 127.0.0.11:53: no such host.
Stacktrace:
   /go/delivery/zeta-node/cmd/zetae2e/local/local.go:182
   /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987
   /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115
   /go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
   /go/delivery/zeta-node/cmd/zetae2e/main.go:16
   /usr/local/go/src/runtime/proc.go:271
   /usr/local/go/src/runtime/asm_arm64.s:1222
e2e setup failed
```

This happens not very often but there might be an easy fix to prevent the issue.

**Comment by lumtis** [comment]
Seems like caused by low disk space
```
INFO [10-04|12:42:58.099]
INFO [10-04|12:42:58.099] Initialising Ethereum protocol           network=1337 dbversion=<nil>
INFO [10-04|12:42:58.099] Loaded most recent local header          number=0 hash=826fe1..9cbaf8 td=1 age=55y6mo2w
INFO [10-04|12:42:58.099] Loaded most recent local full block      number=0 hash=826fe1..9cbaf8 td=1 age=55y6mo2w
INFO [10-04|12:42:58.099] Loaded most recent local fast block      number=0 hash=826fe1..9cbaf8 td=1 age=55y6mo2w
WARN [10-04|12:42:58.099] Failed to load snapshot, regenerating    err="missing or corrupted snapshot"
INFO [10-04|12:42:58.099] Rebuilding state snapshot
INFO [10-04|12:42:58.099] Resuming state snapshot generation       root=0af0d5..dac837 accounts=0 slots=0 storage=0.00B dangling=0 elapsed="128.417µs"
INFO [10-04|12:42:58.099] Regenerated local transaction journal    transactions=0 accounts=0
INFO [10-04|12:42:58.099] Generated state snapshot                 accounts=10 slots=0 storage=412.00B dangling=0 elapsed="325.75µs"
INFO [10-04|12:42:58.100] Gasprice oracle is ignoring threshold set threshold=2
WARN [10-04|12:42:58.100] Error reading unclean shutdown markers   error="leveldb: not found"
WARN [10-04|12:42:58.100] Engine API enabled                       protocol=eth
WARN [10-04|12:42:58.100] Engine API started but chain not configured for merge yet
INFO [10-04|12:42:58.100] Stored checkpoint snapshot to disk       number=0 hash=826fe1..9cbaf8
INFO [10-04|12:42:58.101] Starting peer-to-peer node               instance=Geth/v1.10.26-stable-e5eb32ac/linux-arm64/go1.18.8
WARN [10-04|12:42:58.101] P2P server will be useless, neither dialing nor listening
INFO [10-04|12:42:58.104] IPC endpoint opened                      url=/root/data/geth.ipc
INFO [10-04|12:42:58.104] Generated JWT secret                     path=/root/data/geth/jwtsecret
INFO [10-04|12:42:58.104] HTTP server started                      endpoint=172.20.0.100:8545 auth=false prefix= cors=https://remix.ethereum.org vhosts=*
INFO [10-04|12:42:58.104] WebSocket enabled                        url=ws://127.0.0.1:8551
INFO [10-04|12:42:58.105] HTTP server started                      endpoint=127.0.0.1:8551    auth=true  prefix= cors=localhost                  vhosts=localhost
INFO [10-04|12:42:58.105] Transaction pool price threshold updated price=0
INFO [10-04|12:42:58.105] Updated mining threads                   threads=0
INFO [10-04|12:42:58.105] Transaction pool price threshold updated price=1
INFO [10-04|12:42:58.105] Etherbase automatically configured       address=0xC4AFaD3D1FCB3deb775506E8C8f371e90A5B1Ce6
INFO [10-04|12:42:58.105] Commit new sealing work                  number=1 sealhash=6ec6c6..324b3f uncles=0 txs=0 gas=0 fees=0 elapsed="74.875µs"
INFO [10-04|12:42:58.105] Commit new sealing work                  number=1 sealhash=6ec6c6..324b3f uncles=0 txs=0 gas=0 fees=0 elapsed="186.917µs"
ERROR[10-04|12:42:58.105] Low disk space. Gracefully shutting down Geth to prevent database corruption. available=492.11MiB
INFO [10-04|12:42:58.105] Got interrupt, shutting down...
INFO [10-04|12:42:58.105] New local node record                    seq=1,728,045,778,104 id=7676cc9d1d720d88 ip=127.0.0.1 udp=0 tcp=0
INFO [10-04|12:42:58.105] Started P2P networking                   self=enode://7259b7a1d03486d5f134e5951a3c35f7b279518d0a4f1f2d90f1df9309434aece7cb675caeabfd7d1c797d7f579d2f14acc5a9d31278bd5f6fe745d38f1d3dea@127.0.0.1:0
INFO [10-04|12:42:58.105] Successfully sealed new block            number=1 sealhash=6ec6c6..324b3f hash=2f7d42..97fdb8 elapsed="381.667µs"
INFO [10-04|12:42:58.105] 🔨 mined potential block                  number=1 hash=2f7d42..97fdb8
INFO [10-04|12:42:58.105] Commit new sealing work                  number=2 sealhash=1dda1a..930611 uncles=0 txs=0 gas=0 fees=0 elapsed="136.125µs"
INFO [10-04|12:42:58.105] Commit new sealing work                  number=2 sealhash=1dda1a..930611 uncles=0 txs=0 gas=0 fees=0 elapsed="218.584µs"
WARN [10-04|12:42:58.106] HTTP server graceful shutdown timed out
INFO [10-04|12:42:58.106] HTTP server stopped                      endpoint=172.20.0.100:8545
WARN [10-04|12:42:58.107] HTTP server graceful shutdown timed out
INFO [10-04|12:42:58.107] HTTP server stopped                      endpoint=127.0.0.1:8551
INFO [10-04|12:42:58.107] IPC endpoint closed                      url=/root/data/geth.ipc
INFO [10-04|12:42:58.107] Ethereum protocol stopped
INFO [10-04|12:42:58.107] Transaction pool stopped
INFO [10-04|12:42:58.108] Writing cached state to disk             block=1 hash=2f7d42..97fdb8 root=0af0d5..dac837
INFO [10-04|12:42:58.108] Persisted trie from memory database      nodes=0  size=0.00B   time="1.667µs"  gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [10-04|12:42:58.108] Writing snapshot state to disk           root=0af0d5..dac837
INFO [10-04|12:42:58.108] Persisted trie from memory database      nodes=0  size=0.00B   time=667ns      gcnodes=0 gcsize=0.00B gctime=0s livenodes=1 livesize=0.00B
INFO [10-04|12:42:58.108] Writing clean trie cache to disk         path=/root/data/geth/triecache threads=11
INFO [10-04|12:42:58.108] Persisted the clean trie cache           path=/root/data/geth/triecache elapsed="475.375µs"
INFO [10-04|12:42:58.108] Blockchain stopped
```

**Comment by lumtis** [comment]
Was definitely linked to storage
